### PR TITLE
Create definition/quality/mesh of TiZYX K25

### PR DESCRIPTION
**Pull request by ValentinPitre**

### CODE REVIEW

File: resources/definitions/tizyx_k25.def.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
...red", "ultimaker_abs_silver-metallic", "ultimaker_abs_white", "ultimaker_abs_yellow", "ultimaker_bam", "ultimaker_cpe_black", "ultimaker_cpe_blue", "ultimaker_cpe_dark-grey", "ultimaker_cpe_green", "ultimaker_cpe_light-grey", "ultimaker_cpe_plus_black", "ultimaker_cpe_plus_transparent", "ultimaker_cpe_plus_white", "ultimaker_cpe_red", "ultimaker_cpe_transparent", "ultimaker_cpe_white", "ultimaker_cpe_yellow", "ultimaker_nylon_black", "ultimaker_nylon_transparent", "ultimaker_pc_black", "ultimaker_pc_transparent", "ultimaker_pc_white", "ultimaker_pla_black", "ultimaker_pla_blue", "ultimaker_pla_green", "ultimaker_pla_magenta", "ultimaker_pla_orange", "ultimaker_pla_pearl-white", "ultimaker_pla_red", "ultimaker_pla_silver-metallic", "ultimaker_pla_transparent", "ultimaker_pla_white", "ultimaker_pla_yellow", "ultimaker_pp_transparent", "ultimaker_pva", "ultimaker_tough_pla_black", "ultimaker_tough_pla_green", "ultimaker_tough_pla_red", "ultimaker_tough_pla_white", "ultimaker_tpu_black", "ultimaker_tpu_blue", "ultimaker_tpu_red", "ultimaker_tpu_white", "verbatim_bvoh_175", "Vertex_Delta_ABS", "Vertex_Delta_PET", "Vertex_Delta_PLA", "Vertex_Delta_TPU", "zyyx_pro_flex", "zyyx_pro_pla" ],
```
Comment (diegopradogesto) [inline resources/definitions/tizyx_k25.def.json]: I'm hesitating about this. You are excluding all the materials, so are you only allowing to print with your `tizyx_pla`?

File: resources/definitions/tizyx_k25.def.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        "preferred_material": "tizyx_pla",
```
Comment (diegopradogesto) [inline resources/definitions/tizyx_k25.def.json]: Setting the `tizyx_pla` as the preferred material, without this material being in Cura, makes the material selector to be empty.
image

In that case, you will probably want to add this material to Cura as well.

Comment (fieldOfView) [reply resources/definitions/tizyx_k25.def.json]: See https://github.com/Ultimaker/fdm_materials/pull/39

Comment (diegopradogesto) [reply resources/definitions/tizyx_k25.def.json]: Oh thanks!! I didn't notice it.

**Comment by ValentinPitre** [comment]
Yes I'm allowing only materials by TiZYX which are actually as pull request

**Pull request merged**